index.html

```diff
@@ -0,0 +1,181 @@
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+
+        <img src="images/photo3.png" alt="Third Card">
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+        <img src="images/photo4.png" alt="Forth Card" >
+        <h2 class="head">Multi-post Stories</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> 2015</li>
+          </ul>
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+    </section>
+    <section class="section-about">
+        <div class="about-grid">
+          <div class="text">
+            <div class="headline-text">
+              <h1>About Myself</h1>
+
+              <p class="paragraph1">Hello I’m a software developer! I can help you build a product , feature or website
+                Look through some of my work and experience! If you like what you see and have a
+                project you need coded, don’t hestiate to contact me.</p>
+              <h3 class="lets-connect"> LET'S CONNECT</h3>
+
+              <ul class="buttons getmyr">
+                <li><a href="#"> <img src="./images/twitter.png" alt="social"> </a></li>
+                <li><a href="#"> <img src="./images/Linkedin.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/medium.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/github.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/angellist.png" alt="social"></a></li>

```

Comment:
- [ ] For all social links please add the correct link address of your social profile accounts because you're now building your Portfolio 😉.